doc/mgmt/Docker to Host communication.md

```diff
@@ -251,12 +258,56 @@ N/A
 N/A
 ### 3.6.3 REST API Support
 N/A
+### 3.6.4 D-Bus API
+#### 3.6.4.1 showtech.info
+This API will execute the "show techsupport" command.
+
+Input is date message, and output are return code and error message.
+
+"show techsupport" command could take a few minutes, and D-Bus client should use timeout or asynchronous request.
+
+#### 3.6.4.2 config.save
+This API will execute the "config save" command.
+
+Input is config file name, and output are return code and error message.
+
+#### 3.6.4.3 config.reload
+This API will execute the "config reload" command.
+
+Input is config file name, and output are return code and error message.
+
+"config reload" command would restart D-Bus host service, so D-Bus client might not receive the response.
+
+#### 3.6.4.4 gcu.apply_patch_db
+This API will execute the "config apply-patch" command with SONiC DB schema.
+
+Input is patch file name, and output are return code and error message.
+
+"config apply-patch" command could take a few seconds, and D-Bus client should use timeout or asynchronous request.
+
+#### 3.6.4.5 gcu.apply_patch_yang
+This API will execute the "config apply-patch" command with SONiC Yang schema.
+
+Input is patch file name, and output are return code and error message.
+
+"config apply-patch" command could take a few seconds, and D-Bus client should use timeout or asynchronous request.
+
+#### 3.6.4.6 gcu.create_cp
+This API will execute the "config checkpoint" command.
+
+Input is checkpoint name, and output are return code and error message.
+
+#### 3.6.4.7 gcu.delete_cp
+This API will execute the "config delete-checkpoint" command.
+
+Input is checkpoint name, and output are return code and error message.
 
 # 4 Flow Diagrams
 
 ![](images/docker-to-host-service.svg)
 
 # 5 Error Handling
+If dbus host service exited abnormally, systemd would restart this service.
```

Comment:
What is the client behavior during dbus host service exited abnormally?

Reply:
client will get a dbus error message.